moto/swf/exceptions.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import unicode_literals
+
+from boto.exception import JSONResponseError
+
+
+class SWFClientError(JSONResponseError):
```

Comment:
We've been trying to move to using exceptions so that HTTPretty will return the error code instead of these exceptions jumping up the stack. This allows for the exceptions to work in stand-alone server mode as well. See [here](https://github.com/spulec/moto/blob/master/moto/sns/exceptions.py#L5-L10) for an example.